README.md

```diff
@@ -1,25 +1,19 @@
 # RPM for drush
 
-This repository builds an RPM for drush.
-
-## Relevant drush RPM names
-
-- drush: Legacy RPM containing drush and site-audit
-- drush-0.x: RPM containing only drush (version 0.x)
-- site-audit-0.x: RPM containing only site-audit 0.x
+This repository builds RPMs for drush.
 
 The RPM filename built by this repository is:
 ```
-drush-0.x-release-0.22.0-01458238016.git4602714.x86_64.rpm
+drush-x-release-8.1.2-1466798938.2339402.x86_64.rpm
 {  name  }-{ type}-{vers}-{iteration}.{ commit }.{arch}.rpm
 ```
-The iteration number is the Circle build number for officiel builds, and a timestamp (seconds since the epoch) for locally-produced builds. The build script will refuse to make an RPM when there are uncommitted changes to the working tree, since the commit hash is included in the RPM name.
+The iteration number is the Circle build number for official builds, and a timestamp (seconds since the epoch) for locally-produced builds. The build script will refuse to make an RPM when there are uncommitted changes to the working tree, since the commit hash is included in the RPM name.
 
 ## Install Location
 
 This rpm installs:
 
-/opt/pantheon/drush-0.x/drush.phar
+/opt/pantheon/drush-x/drush-x/drush.phar
```

Comment:
This does not install drush.phar; it installs the entire cloned drush repository, plus the `vendor` directory created from `composer install`, right?